bin/udb3.php

```diff
@@ -132,9 +132,9 @@
 try {
     $consoleApp->run();
 } catch (\Exception $exception) {
-    $app[SentryErrorHandler::class]->handle($exception);
+    $app[ErrorLogger::class]->log($exception);
     $consoleApp->renderException($exception, new ConsoleOutput());
```

Comment:
I now notice that this function is deprecated. Not related to this change but related to the Symfony upgrade, maybe best to use `renderThrowable` now?

Reply:
Yes, that did not exist until we updated Symfony Console recently.

Reply:
Thanks for mentioning it!

Reply:
Ah but we are still on Symfony Console `v3.4.47` and that `renderThrowable` does not exist in that version yet? And `renderException` has been marked as deprecated in `4.x` if I'm not mistaken